src/constants/veNft/veNftResolverAbi.ts

```diff
@@ -0,0 +1,155 @@
+export const veNftResolverAbi = [
```

Comment:
**question** guessing we can eventually get this from an npm package (same as above)?